chihuahua/src/hcall/wasmi.rs

```diff
@@ -634,6 +886,14 @@ impl Chihuahua for WasmiHostProvisioningState {
         self.add_new_data_source(metadata)
     }
 
+    #[inline]
```

Comment:
Missing Rustdoc comments, here (and throughout the remainder of this file).

Reply:
I put documentation in the first line of `impl Chihuahua ......` in a similar pattern as in the wasmtime.rs.